test/integration/test.speech_to_text.js

```diff
@@ -380,5 +380,34 @@ describe('speech_to_text_integration', function() {
       speech_to_text.deleteCustomization({ customization_id: customization_id }, done);
       customization_id = null;
     });
+
+    it('registerCallback()', function(done) {
+      const params = { callback_url: 'http://www.callback.fr/results', user_secret: 'ThisIsMySecret' };
```

Comment:
(If it needs to have a specific response, we can add that with a PR to https://github.com/germanattanasio/watson-test-resources/)